Configuration/TypoScript/Plugins/kitodo.typoscript

```diff
@@ -3,17 +3,17 @@
 # ------------------------------------
 
 # map set[mets] --> tx_dlf[id]
-[like(traverse(request.getQueryParams(), 'set/mets'), 'http*')]
+[request and request.getQueryParams()['set']['mets'] matches '/http.*/']
```

Comment:
```suggestion
[request and like(traverse(request.getQueryParams(), 'set/mets'), 'http*')]
```

Reply:
My original code breaks the normal page view and also several backend menus. Adding `request and ` to the original code here and also below seems to be a better fix.